src/details/owner_borrow.yaml

```diff
@@ -1,5 +1,405 @@
+- title: Understanding ownership
+  shortTitle: ownership
+  key: ownership
+  details: |
+    - Ownership is Rust’s most unique feature, and it enables Rust to make memory safety guarantees without needing a garbage collector.
+    - Rust’s central feature is ownership. All programs have to manage the way they use a computer’s memory while running.
+    - Some languages have garbage collection that constantly looks for no longer used memory as the program runs; in other languages, the programmer must explicitly allocate and free the memory.
```

Comment:
Copying the sentences is not acceptable at all

`All programs have to manage the way they use a computer’s memory while running. Some languages have garbage collection that regularly looks for no-longer used memory as the program runs; in other languages, the programmer must explicitly allocate and free the memory. Rust uses a third approach: memory is managed through a system of ownership with a set of rules that the compiler checks. `
https://doc.rust-lang.org/book/ch04-01-what-is-ownership.html#what-is-ownership